src/app/lists/CommentBox.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import supabase from "@/supabase";
+import { Button } from "@mui/material";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+interface CommentProps {
+    originalComment: string;
+}
+
+const HeaderAndButtonContainer = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+    min-width: 8rem;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: space-between;
+    @media (min-width: 800px) {
+        width: 15%;
+    }
+`;
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    background-color: ${(props) => props.theme.main.background[2]};
+    border-radius: 1rem;
+    margin: 0.5rem;
+    padding: 0.5rem;
+`;
+
+const CommentBoxContainer = styled.div`
+    width: 100%;
+    margin: 1rem;
+    min-width: 15rem;
+    & > * {
+        width: 100%;
+    }
```

Comment:
try and use `display: flex` and `align-items: stretch`, I think that may work

Reply:
Rip forgot flex-direction column